src/output/tree.rs

```diff
@@ -354,7 +360,11 @@ impl<'module> MonoidalPrinter<'module> {
                 }
                 _ => panic!("expected branch, found {value:?}"),
             },
-            Format::Dynamic(_) => self.compile_value(scope, value),
+            Format::Dynamic(name, _dynformat, format) => {
+                let mut child_scope = Scope::child(scope);
+                child_scope.push(name.clone(), Value::Tuple(vec![])); // FIXME?
```

Comment:
Do we have an idea of what should go here, or what we might need to implement to have an appropriate replacement for this scaffolding?

Reply:
For now I don't think we need anything, in future we could potentially save the generated format for optional display, I will update the comment to say that.